test/StreamWrapperTest.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace PhoneBurnerTest\Http\Message;
+
+use PhoneBurnerTest\Http\Message\Fixture\StreamWrapperFixture;
+use Psr\Http\Message\StreamInterface;
+
+class StreamWrapperTest extends WrapperTestCase
+{
+    protected const WRAPPED_CLASS = StreamInterface::class;
+    protected const FIXTURE_CLASS = StreamWrapperFixture::class;
+
+    /**
+     * @test
+     */
+    public function closeIsProxied()
```

Comment:
MUST: Here and elsewhere in the class, declare a return type of `void` on the test methods.

Reply:
I _knew_ I was going to forget to run that inspection.